### PR TITLE
Added README.adoc file for issue Convert Markdown files to AsciiDoc #656

### PR DESCRIPTION
**Pull request by DhirajSardar**
Convert Markdown to AsciiDoc for Issue #656

Converted Markdown files to AsciiDoc format to address issue #656. This PR, #749, transforms the documentation for improved readability and maintainability.

Changes made:
- Converted README.md to AsciiDoc.
- Ensured formatting, code snippets, and links remain accurate.
- Verified cross-references and internal links.

This commit contributes to resolving the issue by aligning documentation with our standards and strategy.

Issue: #656
PR:#749

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+JUnit Pioneer is released on https://github.com/junit-pioneer/junit-pioneer/releases[GitHub] and https://search.maven.org/artifact/org.junit-pioneer/junit-pioneer[Maven Central]. Coordinates:
+
+* group ID: `org.junit-pioneer`
+* artifact ID: `junit-pioneer`
+* version: image::https://maven-badges.herokuapp.com/maven-central/org.junit-pioneer/junit-pioneer/badge.svg?style=flat[Latest Junit Pioneer on Maven Central, link="https://mvnrepository.com/artifact/org.junit-pioneer/junit-pioneer"]
+
+For Maven:
+
+```xml
```
Comment (Michael1993) [inline README.adoc:38]: Please convert to AsciiDoc style code block

**Comment by DhirajSardar** [comment]
Sure

**Comment by DhirajSardar** [comment]
Is there any other things to  be changed ?

**Comment by Michael1993** [comment]
You can also delete the README.md file, since it will be replaced by the .adoc file

**Comment by beatngu13** [comment]
Meanwhile, we converted all Markdown files to Asciidoc (#763). Thank you anyway! 🙏

**Pull request closed without merging**